prow/plugins/cla/cla_test.go

```diff
@@ -140,6 +141,34 @@ func TestCLALabels(t *testing.T) {
 			addedLabels:   []string{fmt.Sprintf("/#3:%s", labels.ClaNo)},
 			removedLabels: []string{fmt.Sprintf("/#3:%s", labels.ClaYes)},
 		},
+		{
+			name:      "EasyCLA status failure removes \"cncf-cla: yes\" label",
+			context:   "EasyCLA",
```

Comment:
More of a nice-to-have cosmetic thing... is `EasyCLA` hardcoded on your end? It would be preferable to have something like `cla/EasyCLA`

Reply:
On the LF side, its hard-coded =/
Here is an example:
https://api.github.com/repos/CNCF-EasyCLAV2-Test/Repository1/statuses/31eb937fb47d258b060a762165f293de6f99d191

Reply:
Just to confirm - so this remains as `EasyCLA` right? And not `cla/EasyCLA`

Reply:
Sure, if it's hardcoded I guess it has to be `EasyCLA`